dragonfly/block/instrument/instrument.go

```diff
@@ -0,0 +1,86 @@
+package instrument
+
+// Instrument represents a note block instrument.
+type Instrument struct {
+	MagicNumber int
```

Comment:
Would be better to wrap this in an unexported `instrument` struct so new Instruments (with invalid values) cannot be constructed.